test/page.spec.js

```diff
@@ -1653,6 +1654,15 @@ module.exports.addTests = function({testRunner, expect, headless}) {
       });
       expect(screenshot).toBeGolden('screenshot-grid-fullpage.png');
     });
+    it('should take fullPage screenshots when defaultViewport is null', async({server}) => {
```

Comment:
Let's move this test to `puppeteer.spec.js`. Tests in `page.spec.js` are getting a pre-launched browser with a page; tests in `puppeteer.spec.js` only get pre-launched HTTP server. (if curious, you can see this setup done in `test/test.js`).